apps/cowswap-frontend/src/modules/injectedWidget/updaters/InjectedWidgetUpdater.tsx

```diff
@@ -105,7 +104,25 @@ export function InjectedWidgetUpdater() {
       stopListeningWindowListener(window, updateParamsListener)
       stopListeningWindowListener(window, updateAppDataListener)
     }
-  }, [updateMetaData, navigate, updateParams, isPartnerFeeEnabled])
+  }, [updateMetaData, navigate, updateParams])
+
+  // Log an error when partnerFee was set and then discarded
+  useEffect(() => {
+    if (prevPartnerFee || !partnerFee) {
+      const sentryError = Object.assign(
+        new Error(`BPS: ${prevPartnerFee?.bps}, recipient: ${prevPartnerFee?.recipient}`),
+        {
+          name: 'PartnerFeeDiscarded',
+        }
+      )
+
+      Sentry.captureException(sentryError, {
+        tags: {
+          errorType: 'PartnerFeeDiscarded',
+        },
+      })
+    }
```

Comment:
your alert worked, we see some errors now on sentry. Are we closer to understanding the issue?